src/httpserver/http_request.hpp

```diff
@@ -442,7 +444,7 @@ class http_request
         **/
         void set_arg(const std::string& key, const std::string& value)
         {
-            this->args[key] = value;
+            this->args[key] = value.substr(0,content_size_limit);
```

Comment:
I see.

I think, that having this kind of semantic, a utility such as the one you are describing could be very useful.